src/main/java/com/fabriccommunity/spookytime/world/feature/RandomChoiceSelector.java

```diff
@@ -0,0 +1,11 @@
+package com.fabriccommunity.spookytime.world.feature;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.Random;
+
+@FunctionalInterface
+public interface RandomChoiceSelector {
```

Comment:
BlockStateSelector?
RandomBlockSelector?
Maybe say what you're choosing from a little more specifically in the name of it